db/models.py

```diff
@@ -50,3 +53,60 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(on_delete=CASCADE, to="User")
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+    class Meta:
+        ordering = ("-created_at",)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(to=MovieSession, on_delete=CASCADE)
+    order = models.ForeignKey(to=Order, on_delete=CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=[
+                    "movie_session",
+                    "row",
+                    "seat"
+                ], name="unique_seat_per_session"
+            )
+        ]
+
+    def clean(self) -> None:
+        if self.row < 1 or self.row > 10:
+            raise ValidationError({
+                "row":
+                    "row number must be in available range: (1, rows): (1, 10)"
+            })
+        if self.seat < 1 or self.seat > 12:
+            raise ValidationError({
+                "seat": (
+                    "seat number must be in available range: "
+                    "(1, seats_in_row): (1, 12)"
+                )
+            })
```

Comment:
The number of rows and seats are hard-coded in the `Ticket` model. This is not a good practice as the number of rows and seats can vary from one cinema hall to another. Consider fetching the number of rows and seats from the associated `CinemaHall` instance.